programs/sbf/Cargo.toml

```diff
@@ -80,6 +80,7 @@ walkdir = "2"
 bincode = { workspace = true }
 byteorder = { workspace = true }
 elf = { workspace = true }
+indexmap = { workspace = true }
```

Comment:
@brooksprumo: This error really got me 😭 .  Do you happen to know why I can't specify its version from the workspace while I see the root Cargo.toml does have a row for `indexmap`?


```

error: failed to parse manifest at `/var/lib/buildkite-agent/builds/ci5/solana-labs/solana/programs/sbf/Cargo.toml`
--
  |  
  | Caused by:
  | error inheriting `indexmap` from workspace root manifest's `workspace.dependencies.indexmap`
  |  
  | Caused by:
  | `dependency.indexmap` was not found in `workspace.dependencies`


```